From: Ralf Treinen <treinen@debian.org>
Date: Tue, 19 May 2009 17:28:09 +0200
Subject: Patch config.sh for installation

Use for installatation a patched config.sh (created by debian/rules).
---
 build/install.sh         |    2 +-
 build/partial-install.sh |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/install.sh b/build/install.sh
index df01db4..546d65a 100755
--- a/build/install.sh
+++ b/build/install.sh
@@ -18,7 +18,7 @@ set -e
 
 cd `dirname $0`/..
 
-. config/config.sh
+. config/config.debian.install.sh
 
 not_installed=$PWD/_build/not_installed
 
diff --git a/build/partial-install.sh b/build/partial-install.sh
index a8113c9..15071d4 100755
--- a/build/partial-install.sh
+++ b/build/partial-install.sh
@@ -22,7 +22,7 @@ set -e
 
 cd `dirname $0`/..
 
-. config/config.sh
+. config/config.debian.install.sh
 
 not_installed=$PWD/_build/not_installed
 
-- 
